src/runtime/Context.cpp

```diff
@@ -119,7 +119,10 @@ void Context::printDebugger(StringView* output)
     if (!m_debugger || !m_debugger->enabled()) {
         return;
     }
-    m_debugger->sendString(Debugger::ESCARGOT_MESSAGE_PRINT_8BIT, output);
+    m_debugger->sendType(Debugger::ESCARGOT_MESSAGE_PRINT);
+    if (m_debugger->enabled()) {
```

Comment:
Check for enabled debugger is already done in the above.
So please remove this if statement.

Reply:
Unfortunately during a `send` the connection can be dropped, so we need to check after each send whether we still have connection.

This could be reworked to return a true if sending is successful, or sending data does nothing if no connection is available. I think we should do it in a separate patch though.

Reply:
IMHO the latter approach in which `send` checks the connection before actually sending the data looks better. In this approach, we don't need to check the enabled debugger everytime when sending the data.